src/scancode/utils.py

```diff
@@ -59,13 +59,21 @@ class BaseCommand(click.Command):
     # "Try 'scancode --help' for help on options and arguments."
     short_usage_help = ''
 
+    options = []
```

Comment:
I think this would be best only in ScanCommand if this is needed in anyway. Let's leave the extraction out of this entirely for now.